File: backend/components/metiers/threed/legacy/samples/rbac/rbac_sample.js
// rbac_sample.js – Exemple de RBAC ultra avancé
/**
 * Simule un contrôle RBAC legacy Threed
 */
function rbacSample(user, role) {
  // ... logique RBAC avancée ...
  return role === 'admin' ? `${user} a tous les droits` : `${user} accès restreint`;
}

module.exports = rbacSample;

File: backend/components/metiers/threed/legacy/samples/rbac/rbac_sample.py
# rbac_sample.py – Exemple de RBAC ultra avancé


def rbac_sample(user, role):
    """Simule un contrôle RBAC legacy Threed"""
    # ... logique RBAC avancée ...
    return (
        f"{user} a tous les droits"
        if role == "admin"
        else f"{user} accès restreint"
    )
